AI/finder/financial_eval.py

```
@@ -0,0 +1,229 @@
+import re
+from datetime import datetime
+
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
+
+import psycopg2
+
+conn = psycopg2.connect(
+    dbname="neondb",
+    user="neondb_owner",
+    password="npg_hWkg04MwGlYs",
```

Ruff (0.13.3)

Possible hardcoded password assigned to argument: "password"

(S106)